conda.recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
+npm install .
+npm run build
+python setup.py install
+python -m nbpresent.install --force --enable
```

Comment:
> Sorry, install-time. I'm thinking `[+user]` vs `[+global]`.

Not sure there are runtime env variables available for conda... if I have to bet, I would say it does not exist... but maybe you surprise me :wink: